partner_aging/wizard/customer.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import tools
+from odoo import api, fields, models, _
+
+
+class PartnerAgingDate(models.TransientModel):
```

Comment:
[Split models by file](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming)

Reply:
Also this is a wizard, so it should be in the `wizards` folder